src/components/Switch/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+import { BaseProps } from '../types';
+import { SwitchWrapper, StyledSwitch, SwitchText } from './styled';
+import { COLOR_BRAND, COLOR_GRAY_2 } from '../../styles/colors';
+import RenderIf from '../RenderIf';
+
+interface Props extends BaseProps {
+    label?: string | ReactNode;
```

Comment:
If you click on ReactNode you will see what types conform it, here use only ReactNode since string is contained inside it.
```suggestion
    label?: ReactNode;
```
In the case of the proptypes we use string | node to improve documentation because the consumer may not know that node contain string